src/main/java/com/aws/greengrass/shadowmanager/ShadowManager.java

```diff
@@ -102,6 +102,7 @@ public class ShadowManager extends PluginService {
     private final CloudDataClient cloudDataClient;
     private final MqttClient mqttClient;
     private final PubSubIntegrator pubSubIntegrator;
+    private final AtomicReference<Strategy> currentStrategy = new AtomicReference<>(DEFAULT_STRATEGY);
```

Comment:
this fixes the issue but lets add this to the list of things to refactor - we've got the state not copied in ShadowManager and SyncHandler